pyodide_lock/utils.py

```diff
@@ -74,3 +87,54 @@ def _generate_package_hash(full_path: Path) -> str:
         while chunk := f.read(4096):
             sha256_hash.update(chunk)
     return sha256_hash.hexdigest()
+
+
+def get_wheel_dependencies(wheel_path: Path, pkg_name: str) -> list[Requirement]:
```

Comment:
I think we can parse them with [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html#distribution-metadata) so we don't have to bother checking it's correct for all edge cases.